decidim-assemblies/app/controllers/decidim/assemblies/admin/assembly_members_controller.rb

```diff
@@ -75,6 +75,24 @@ def destroy
           end
         end
 
+        def users
+          respond_to do |format|
+            format.json do
+              render json: [] && return if params.fetch(:term, "").empty?
+
+              query = current_organization.users&.order(name: :asc)
+              term = params[:term]
+              if term&.start_with?("@")
```

Comment:
I've seen this code in other PRs, should we extract this to a class or something?

Reply:
Yo've seen it in the same module? If not, we need to extract it to the `core` module so we don't add a dependency to the `admin`, right? We can extract it to a `query` class?

Reply:
@oriolgual, you refer to this other PR that we are "arguing" there. https://github.com/decidim/decidim/pull/3136#discussion_r185710826

We can look for a solution for both systems. What do you recommend?
Create a `OrganizersController` or `UsersController` but should be done in `Decidim::Core` no?

Reply:
Yes, should be done at core, no problem. I'd go with a `OrganizersController`

Reply:
good! But in this case, will be `UsersController` ?